src/main/java/seedu/duke/Parser.java

```diff
@@ -70,12 +70,30 @@ public static Command checkAction(String inputCommand) throws DukeExceptions {
                 commandType = new TaskSelectCommand(taskIndex, projectIndex);
             }
             break;
+        case "description":
+            if (isProjectListView) {
+                System.out.println("Not in Task View!");
+            } else {
+                for (int i = 1; i < inputs.length; i++) {
+                    if (i == inputs.length - 1) {
+                        description += inputs[i];
+                    } else {
+                        description += inputs[i] + " ";
+                    }
+                }
+                commandType = new ProjectDescriptionCommand(description, projectIndex);
+            }
+            break;
         case "project":
             if (isProjectListView) {
                 for (int i = 1; i < inputs.length; i++) {
-                    projectDescription += inputs[i];
+                    if (i == inputs.length - 1) {
+                        description += inputs[i];
+                    } else {
+                        description += inputs[i] + " ";
+                    }
                 }
-                commandType = new ProjectCommand(projectDescription);
+                commandType = new ProjectCommand(description);
```

Comment:
I feel like we shouldn't use variable "description" for both project name and description as it will be confusing to use one variable for two different purposes.

Reply:
yup agreed